.github/workflows/containers.yml

```
@@ -0,0 +1,203 @@
+
+      - id: set-deploy-matrix-count
+        run: echo "value=$( echo '${{ steps.set-deploy-matrix.outputs.value }}' | jq '.path | length')" >> $GITHUB_OUTPUT
+
+  test:
+    name: Run Tests
+    needs: set-matrices
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: ${{fromJson(needs.set-matrices.outputs.build-matrix)}}
+    steps:
+      - uses: actions/checkout@v2
```

actionlint (1.7.4)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)